src/OpenSage.Game/Data/Ini/AudioSettings.cs

```diff
@@ -196,5 +207,31 @@ public sealed class AudioSettings
 
         [AddedIn(SageGame.Bfme)]
         public float LivingWorldZoomMaxDistance { get; private set; }
+
+        //BFME2
```

Comment:
In general you don't need this comment, because the information is already in the per-property attributes.

Reply:
Yep, i just added it as a comment to myself.